### ISSUE TITLE
Scylla fails to install on Debian 10 because of dependency to OpenJDK 8

### ISSUE THREAD

**Issue by penberg** [scylladb/scylla#7207]
Installation of Scylla 4.1 fails on Debian 10 as follows:

```
root@35bece632d5c:/# apt-get install scylla
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 scylla : Depends: scylla-jmx but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

which is because `scylla-jmx` requires `openjdk8`:

```
root@35bece632d5c:/# apt-get install scylla-jmx
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 scylla-jmx : Depends: openjdk-8-jre-headless but it is not installable or
                       openjdk-8-jre but it is not installable or
                       oracle-java8-set-default but it is not installable or
                       adoptopenjdk-8-hotspot-jre but it is not installable
E: Unable to correct problems, you have held broken packages.
```

The issue is fixed in https://github.com/scylladb/scylla-jmx/commit/d5d1efd1887e9d976e8eeaba59684aa6fab2719c but the commit needs to be backported.

**Comment by penberg** [comment]
I backported the fix to 4.2.

**Comment by slivne** [comment]
@penberg - thanks - this is resolved, right - closing assuming it is - if not please reopen

fixed by ade4edaa20349c84cacd70e8caec1764986ed96d

**Comment by penberg** [comment]
Yes, the backport to 4.2 is indeed https://github.com/scylladb/scylla-jmx/commit/ade4edaa20349c84cacd70e8caec1764986ed96d (in `scylla-jmx.git`).
